functions/src/modules/reporting.sync.functions.ts

```diff
@@ -0,0 +1,82 @@
+import * as functions from 'firebase-functions';
+import * as  admin from 'firebase-admin';
+
+const bucket = admin.storage().bucket();
+const db = admin.database();
+import * as helpers from './../helpers';
+
+/////////////////////////////////////////////////
+//
+//   Add MP3 name to DB (Storage Upload Trigger)
+//
+/////////////////////////////////////////////////
+
+export const importFilesFromStorage = functions.storage.object()
+.onFinalize( object => {
+    const filePath = object.name;
+    if (!helpers.checkValidMP3(filePath))
+        return 1;
+    
+    let mpo3Ref = db.ref(helpers.createMP3DBRef(filePath));
```

Comment:
typo

Reply:
Moreover, mp3 is not mentioned in the database, so really this is a `fileRef`.

Reply:
And I think creating the ref can be moved from helpers here. It will hardly be used anywhere else.